README.md

```diff
@@ -1,104 +1,128 @@
-# pravega-samples
+# Pravega and Analytics Connectors Examples
 
-Sample applications for Pravega.
+This repository contains code examples to demonstrate how developers can work with 
+[Pravega](http://pravega.io). We also provide code examples to connect analytics 
+engines such as [Flink](https://flink.apache.org/) and
+[Hadoop](http://hadoop.apache.org/) with Pravega as a storage substrate for data 
+streams. 
 
-## Getting Started
+For more information on Pravega, we recommend to read the [documentation and the
+developer guide](http://pravega.io).
 
-### Building Pravega
+# Repository Structure
 
-Optional: This step is required only if you want to use a different version
-of Pravega than is published to maven central.
+This repository is divided into sub-projects specifically addressed to demonstrate a
+single component (`standalone-examples` and `flink-examples`). 
```

Comment:
Maybe 
This repository is divided into sub-projects (`standalone-examples` and `flink-examples`)  specifically addressed to demonstrate a single component.
 ?

Reply:
Done.